tss-esapi/src/context/tpm_commands/signing_and_signature_verification.rs

```diff
@@ -45,9 +45,10 @@ impl Context {
         key_handle: KeyHandle,
         digest: Digest,
         scheme: SignatureScheme,
-        validation: HashcheckTicket,
+        validation: Option<HashcheckTicket>,
```

Comment:
I think it looks good but just for the sake of completeness, to avoid API breaks you could make `validation: impl Into<Option<HashcheckTicket>>` and below add a call to `.into()` before `.unwrap_or_default()`.

This would allow clients to call this function with `HashcheckTickets` like previously (no modification needed, no Some wrapping) but would also accept `None`.

An example: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f509608362b096375d913dbed0a29a88

Reply:
That is neat I added it.